src/System.IO.Compression/tests/Configurations.props

```diff
@@ -4,6 +4,8 @@
     <BuildConfigurations>
       netstandard-Unix;
       netstandard-Windows_NT;
+      netcoreapp-Unix;
```

Comment:
I'm not certain. As netstandard should be a subset of netcoreapp, it wouldn't make much sense to run a netstandard build alongside a netcoreapp build. My assumption is that when running tests against netcoreapp via build-tests.cmd, only the most specific Configuration is ran e.g. netcoreapp if available otherwise netstandard, or netfx where available otherwise netstandard. @karajas probably knows more.

Regardless, the important bit is that the new tests will run only on netcoreapp.